app/models/mixins/authentication_mixin.rb

```diff
@@ -12,6 +12,33 @@ def self.authentication_check_schedule
       assocs = zone.respond_to?(assoc) ? zone.send(assoc) : []
       assocs.each { |a| a.authentication_check_types_queue(:attempt => 1) }
     end
+
+    def self.queue_authentication_check(args, user_id, zone)
```

Comment:
@dclarizio @jntullo I looked into this further, and this is what I think will happen if we go ahead with the queued validation approach...

The UI currently displays a spinner (and stays blocked) while it waits for the validation to return with a success/failure code.

However, based on what's implemented in the UI counterpart of this PR: https://github.com/ManageIQ/manageiq-ui-classic/pull/1580/files#diff-f402e411cd6dc158c724b14d72ead612R74, if we do go ahead with this queued approach, the UI will get a return code from the backend about the queueing operation and __*not the result of the actual validation itself*.__
A successful queued operation will be construed as a Success in the angular code, which will enable the Save button, and technically a provider record *can* be saved at this point.

Question is -- do we want that to happen?
And I think the answer is No, since it opens up other issues related to the state of the workers and such.

Just to be clear, as a short term solution, I'm OK blocking the UI, but the expectation would be that we get the result of the actual validation and not the result of the queued validation operation.